lib/assets/javascripts/cartodb/old_common/utils.js

```diff
@@ -293,3 +293,13 @@
     var value = object[property];
     return _.isFunction(value) ? value.apply(object, Array.prototype.slice.call(arguments, 2)) : value;
   };
+
+
+  /*  Titlecase title function used with aliasing throughout. Various model inputs (vis, table, model)
+      may be appropriate */
+  cdb.Utils.titlecaseName = function(model) {
```

Comment:
I think "title case" is two words and that this should be reflected in the camel-case.